### ISSUE TITLE
[QUESTION]-Missing Parent Records when mapping objects?

### ISSUE THREAD

**Issue by zdooder** [forcedotcom/SFDX-Data-Move-Utility#581]
**Describe the question**
When migrating Opportunities in source org to Orders in target org, SFDMU reports missing parent IDs for the Pricebook2 object.  It is not outputting Target CSV files for the Pricebook2 objects, but I can see in the binary file cache that it is querying them.

I'm sure that I'm just doing something wrong, but I can't see what it is.

**To Reproduce**
Migrate Opportunity objects to Order objects, mapping fields as required.

**Expected behavior**
Pricebook2Id in Order objects should be found in the queried Pricebook2 objects.

**export.json**
[export.json.gz](https://github.com/forcedotcom/SFDX-Data-Move-Utility/files/11034768/export.json.gz)

**Log file**
[2023-03-21__13_17_16.log.gz](https://github.com/forcedotcom/SFDX-Data-Move-Utility/files/11034769/2023-03-21__13_17_16.log.gz)

**MissingParentRecordsReport.csv**
[MissingParentRecordsReport.csv.gz](https://github.com/forcedotcom/SFDX-Data-Move-Utility/files/11034776/MissingParentRecordsReport.csv.gz)

**Comment by hknokh** [comment]
Hello!
Thank you for reaching out to me.
This case seems not to be a bug but mostly a configuration issue. Currently I can handle only cases caused by runtime errors. Please see the [Support Policy](https://help.sfdmu.com/full-documentation/additional-information/support_policy).
Thanks.

**Comment by zdooder** [comment]
In case it helps other people later, I eventually figured out that this was a problem with my configuration.  I had assumed incorrectly that a default "operation" would apply to each of the objects in the configuration.  Once I corrected this, I was able to move past this problem.

It might be nice if SFDMU threw an error if _mandatory_ fields like "operation" are not included.  It would have made it obvious what was wrong; but either way this was definitely user-error.  :)
